src/BLSRegistryCoordinatorWithIndices.sol

```diff
@@ -289,6 +289,66 @@ contract BLSRegistryCoordinatorWithIndices is EIP712, Initializable, IBLSRegistr
         }
     }
 
+    /**
+     * @notice Updates the stakes of all operators for each of the specified quorums in the StakeRegistry. Each quorum also
+     * has their StakeRegistry.quorumTimestamp updated. which is meant to keep track of when operators were last all updated at once. 
+     * @param operatorsPerQuorum is an array of arrays of operators to update for each quorum. Note that each nested array
+     * of operators must be sorted in ascending operatorId order to ensure that all operators in the quorum are updated
+     * @param quorumNumbers is an array of quorum numbers to update
+     * @dev This method is used to update the stakes of all operators in a quorum at once, rather than individually
+     * 
+     */
+    function updateOperatorsPerQuorum(address[][] calldata operatorsPerQuorum, bytes calldata quorumNumbers) external onlyWhenNotPaused(PAUSED_UPDATE_OPERATOR) {
+        require(
+            operatorsPerQuorum.length == quorumNumbers.length,
+            "BLSRegistryCoordinatorWithIndices.updateOperatorsPerQuorum: input length mismatch"
+        );
+        for (uint256 i = 0; i < quorumNumbers.length; ++i) {
+            uint8 quorumNumber = uint8(quorumNumbers[i]);
+            address[] memory currQuorumOperators = operatorsPerQuorum[i];
+            require(
+                currQuorumOperators.length == indexRegistry.totalOperatorsForQuorum(quorumNumber),
+                "BLSRegistryCoordinatorWithIndices.updateOperatorsPerQuorum: number of updated operators does not match quorum total"
+            );
+            bytes32 prevOperatorId = bytes32(0);
+
+            // Update stakes for each operator in this quorum
+            for (uint256 j = 0; j < currQuorumOperators.length; ++j) {
+                address operator = currQuorumOperators[j];
+                bytes32 operatorId = getOperatorId(operator);
+                uint192 currentBitmap = _currentOperatorBitmap(operatorId);
+
+                /**
+                 * Assuming the operatorId cannot be 0, for the first operatorId, prevOperatorId will be 0
+                 * Require check is to prevent duplicate operators and that all quorum operators are updated
+                 */
+                require(
+                    operatorId > prevOperatorId,
```

Comment:
this is kind of confusing for the caller, could we make sure the operatorAddresses are in order instead?

Reply:
Changed https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/99e543c4449e67f25ff3f94eed5d36f4e8099379